packages/backend/src/users/queries/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * TikTok clone as web application
+ * Copyright (C) 2022  Gabrielle Guimarães de Oliveira
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+export * from './currentUserQuery';
```

Comment:
prefer to import files directly, avoid index

![image](https://user-images.githubusercontent.com/29582786/179424921-9d36bd60-d719-4b6c-a571-b3f77d01d028.png)